test/prototype/test_transforms.py

```diff
@@ -256,3 +257,26 @@ def test_mask_transform_mask_bos(self):
             masked_tokens, _, _ = mask_transform(self.sample_token_ids)
             exp_tokens = torch.tensor([[5, 5, 5, 5, 5], [5, 5, 5, 4, 4]])
             self.assertEqual(exp_tokens, masked_tokens)
+
+    def _t5tokenizer(self, test_scripting):
+        asset_name = "https://huggingface.co/t5-base/resolve/main/spiece.model"
+        asset_path = get_asset_path(asset_name)
+        transform = T5Transform(asset_path, max_seq_len=512, eos_idx=1, padding_idx=0)
+        if test_scripting:
+            transform = torch.jit.script(transform)
+
+        actual = transform(["Hello World!, how are you?"])
+        expected = torch.tensor([[8774, 1150, 55, 6, 149, 33, 25, 58, 1]])
+        self.assertEqual(actual, expected)
+
+        actual = transform("Hello World!, how are you?")
+        expected = torch.tensor([8774, 1150, 55, 6, 149, 33, 25, 58, 1])
+        self.assertEqual(actual, expected)
```

Comment:
Are there any differences between these 2 test cases?

Reply:
yeah, in the first case the input is passed in as a list (format for handling batched sequences) vs the input for the second case is just a single string. I can make the first test more obviously batched by adding a second sequence if that's easier to follow.